PowerEditor/installer/APIs/cs.xml

```diff
@@ -16,8 +17,205 @@
 		<KeyWord name="class" />
 		<KeyWord name="Comparer" />
 		<KeyWord name="Console" />
-		<KeyWord name="Console.Write(" />
-		<KeyWord name="Console.WriteLine(" />
+		<KeyWord name="Console.Write" func="yes">
+			<Overload retVal="void" >
+				<Param name="char[] buffer, int index, int count" />

```

Comment:
Each parameter needs its own `<Param>` element

Reply:
Okk. I got it now. I will make required changes to the file.